### PR TITLE
tempest compute features

### PR DESCRIPTION
**Pull request by SeanMooney**
- configure tempest to disable ide_bus tests
- enable BFV migration tests

### CODE REVIEW

**Review by gibizer — APPROVED** [review]
This is test only change so I'm OK to land it even during beta freeze.

**Comment by gibizer** [comment]
/test precommit-check

```
* 2024-05-02T16:29:03Z 5x default-scheduler: 0/100 nodes are available: 19 node(s) were unschedulable, 2 Insufficient memory, 24 node(s) had untolerated taint {node-role.kubernetes.io/ci-builds-worker: ci-builds-worker}, 3 node(s) had untolerated taint {node-role.kubernetes.io/infra: }, 3 node(s) had untolerated taint {node-role.kubernetes.io/master: }, 33 Insufficient cpu, 4 node(s) had untolerated taint {node-role.kubernetes.io/ci-longtests-worker: ci-longtests-worker}, 6 node(s) had untolerated taint {node-role.kubernetes.io/ci-prowjobs-worker: ci-prowjobs-worker}, 8 node(s) didn't match Pod's node affinity/selector. no new claims to deallocate, preemption: not eligible due to preemptionPolicy=Never.
```

**Comment by gibizer** [comment]
multiattach does not seems to work https://logserver.rdoproject.org/53/753/7691cb959b2ce04e3e07cab8d42ac997a409fdb5/github-check/nova-operator-tempest-multinode-ceph/61b2b23/controller/ci-framework-data/tests/test_operator/tempest-tests/stestr_results.html

**Comment by SeanMooney** [comment]
ya which is odd
i think it should because ceph support it but for now i might just turn that off and file a bug
the implant thing i BFV livemigration does work which is what was reported as not working.

File: .zuul.yaml [hunk 1]
```diff
@@ -147,7 +148,11 @@
             compute-feature-enabled.stable_rescue true
             compute-feature-enabled.hostname_fqdn_sanitization true
             compute-feature-enabled.live_migration true
+            compute-feature-enabled.live_migrate_back_and_forth true
             compute-feature-enabled.block_migration_for_live_migration false
+            compute-feature-enabled.ide_bus false
+            compute-feature-enabled.volume_backed_live_migration true
+            compute-feature-enabled.volume_multiattach true
```
Comment (SeanMooney) [inline .zuul.yaml:155]: so multi attach requires that i create a new volume type with multi attach enabled

and configure 
https://github.com/openstack/tempest/blob/ccd034eb3a916ce4d3955fa11fe905aa50cbc15d/tempest/config.py#L970-L973

in the tempest config to use it.

i can do the volume conrguration in the pre tempest hook which is currently unused.

https://github.com/openstack-k8s-operators/nova-operator/blob/main/ci/nova-operator-tempest-multinode-ceph/ci_fw_vars.yaml#L27-L34

Comment (SeanMooney) [reply .zuul.yaml:155]: i also think https://github.com/openstack/tempest/blob/ccd034eb3a916ce4d3955fa11fe905aa50cbc15d/tempest/config.py#L1057-L1064 might work with ceph.

File: ci/nova-operator-tempest-multinode-ceph/ci_fw_vars.yaml [hunk 2]
```diff
@@ -27,10 +27,11 @@ post_deploy:
 # we can use this hook to create flavors, images, etc
 # before we execute tempest simiilar to how we would use
 # local.sh in devstack based jobs
-# pre_tempest:
-#   - name: Create manila resources
-#     type: playbook
-#     source: manila_create_default_resources.yml
+pre_tempest:
```
Comment (tosky) [inline ci/nova-operator-tempest-multinode-ceph/ci_fw_vars.yaml:30]: I think pre_tempest may not be working anymore at this point (despite some compatibility code) and you may need to use pre_tests instead

Comment (SeanMooney) [reply ci/nova-operator-tempest-multinode-ceph/ci_fw_vars.yaml:30]: ya i think so too.
ill update the hook I'm using thanks

**Comment by SeanMooney** [comment]
check-rdo

**Comment by SeanMooney** [comment]
check-rdo

**Review by gibizer — APPROVED** [review]
CI is green and this only changes zuul config so we can land this even during the beta freeze

**Pull request merged**